edx_oauth_client/pipeline.py

```diff
@@ -38,13 +38,18 @@ def ensure_user_information(
 
         # Received fields could be pretty different from the expected, mandatory are only 'username' and 'email'
         data['username'] = user_data.get('username', user_data.get('name'))
-        data['first_name'] = user_data.get('firstName', user_data.get('first_name'))
-        data['last_name'] = user_data.get('lastName', user_data.get('last_name'))
+        data['first_name'] = user_data.get('firstname', user_data.get('first_name'))
+        data['second_name'] = user_data.get('secondname', '')
```

Comment:
Did not get the case here `user_data.get('secondname', '')` and the line above you have this `user_data.get('firstname', user_data.get('first_name'))`
As I see it should be consistent thus `user_data.get('secondname', user_data.get('second_name'))`